modules/services/compose/src/main/java/au/com/shiftyjelly/pocketcasts/compose/buttons/TimePlayButton.kt

```diff
@@ -29,36 +29,70 @@ import au.com.shiftyjelly.pocketcasts.localization.R
 import au.com.shiftyjelly.pocketcasts.localization.helper.TimeHelper
 import au.com.shiftyjelly.pocketcasts.ui.theme.Theme
 import com.airbnb.android.showkase.annotation.ShowkaseComposable
+import au.com.shiftyjelly.pocketcasts.images.R as IR
 
 sealed class TimePlayButtonStyle {
     object Solid : TimePlayButtonStyle()
     object Outlined : TimePlayButtonStyle()
 }
 
+sealed class TimePlayButtonColors {
+    @Composable
+    abstract fun backgroundColor(): Color
+
+    @Composable
+    abstract fun textColor(): Color
+
+    @Composable
+    abstract fun borderColor(): Color
+
+    data class Player(
+        val textColor: Color,
+    ) : TimePlayButtonColors() {
+        @Composable
+        override fun backgroundColor() = MaterialTheme.theme.colors.playerContrast01
+
+        @Composable
+        override fun textColor() = textColor
+
+        @Composable
+        override fun borderColor() = textColor
+    }
+
+    object Default : TimePlayButtonColors() {
+        @Composable
+        override fun backgroundColor() = Color.Transparent
+
+        @Composable
+        override fun textColor() = MaterialTheme.theme.colors.primaryText01
+
+        @Composable
+        override fun borderColor() = MaterialTheme.theme.colors.primaryText01
+    }
+}
+
 @Composable
 fun TimePlayButton(
     timeSecs: Int,
     @StringRes contentDescriptionId: Int,
     onClick: () -> Unit,
     modifier: Modifier = Modifier,
-    textColor: Color = MaterialTheme.theme.colors.primaryText01,
-    backgroundColor: Color = MaterialTheme.colors.surface,
-    borderColor: Color = MaterialTheme.theme.colors.primaryText01,
-    buttonStyle: TimePlayButtonStyle = TimePlayButtonStyle.Outlined
+    colors: TimePlayButtonColors = TimePlayButtonColors.Default,
```

Comment:
I combined all colors into `TimePlayButtonColors` just like the compose `ButtonColors` and included predefined colors: `Default` and `Player` just like compose `DefaultButtonColors`.

```
private class DefaultButtonColors(
    private val backgroundColor: Color,
    private val contentColor: Color,
    private val disabledBackgroundColor: Color,
    private val disabledContentColor: Color
) : ButtonColors
```

I did that to other bookmark views and rows for theming.